client/app/dedicatedUniverseComponents/notification/index.js

```diff
@@ -0,0 +1,15 @@
+import angular from 'angular';
+
+import 'ovh-angular-user-pref';
+
+import ducNotification from './notification.service';
```

Comment:
As a service can be instanciable: 

```suggestion
import DucNotification from './notification.service';
```